klasses/views.py

```diff
@@ -0,0 +1,21 @@
+"""views for bootcamps"""
+from django.db.models.functions import Now
+from rest_framework.mixins import ListModelMixin
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.viewsets import GenericViewSet
+
+from klasses.serializers import BootcampRunSerializer
+from klasses.models import BootcampRun
+
+
+class BootcampViewSet(ListModelMixin, GenericViewSet):
+    """Viewset for bootcamps"""
+    permission_classes = (IsAuthenticated,)
+    serializer_class = BootcampRunSerializer
+
+    def get_queryset(self):
+        """Make a queryset which optionally shows what runs are available for enrollment"""
+        queryset = BootcampRun.objects.all().select_related("bootcamp")
+        if self.request.query_params.get("available") == "true":
+            queryset = queryset.filter(start_date__lt=Now()).exclude(enrollments__user=self.request.user)
```

Comment:
I think we'll also want to filter out bootcamps runs for which the user already has an application, active or complete/rejected. We might just want to remove the enrollments check from the query, but I'll leave that up to you

Reply:
I removed the enrollments check and added an application check in its place. It looks like all application states are either awaiting or complete so I think we can just check for the presence of an application.